.pre-commit-config-cpp.yaml

```diff
@@ -2,23 +2,31 @@
 # hooks require tools that may not be trivial to install (unlike the Python
 # hooks like flake8), we keep these in a separate file to be applied manually.
 repos:
--   repo: https://github.com/pocc/pre-commit-hooks
-    rev: python
+-   repo: local
     hooks:
       - id: clang-format
+        name: clang-format
+        entry: clang-format
+        language: system
+        types: [c++]
+        # Ignore external files. Also include AABB.h because it uses intrinsics
+        # heavily and these are not considered portable.
+        exclude: 'extern/.*'
         # Unclear if this is necessary, it seems like clang-format uses
         # .clang-format if found by default, at least for newer versions. I'll
         # specify this here just in case.
         args:
         - --style=file
-        exclude: 'extern/.*'
       - id: clang-tidy
-        # Ignore external files and those taken verbatim from HOOMD.
+        name: clang-tidy
+        entry: clang-tidy
+        language: system
+        types: [c++]
+        # Ignore external files. Also include AABB.h because it uses intrinsics
+        # heavily and these are not considered portable.
```

Comment:
```suggestion
        # Exclude external files. Also exclude AABB.h because it uses intrinsics
        # heavily and these are not considered portable.
```